src/dataSources/DataSourceOrigin.js

```diff
@@ -1,7 +1,6 @@
 'use strict';
 
-var DataSourceBase = require('./DataSourceBase');
-var headerify = require('../Shared.js').analytics.util.headerify;
+var DataSourceBase = require('fin-hypergrid-data-source-base');
```

Comment:
By the way (not part of these requested changes): We need to remove the `getFields` calls from fin-hypergrid-data-source-base.